docs/internals/generic/_index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Generic Service"
+linkTitle: "Generic"
```

Comment:
Actually I think this needs to the full name otherwise it looks a bit odd 
```suggestion
linkTitle: "Generic Service"
```

Reply:
Also wondering if this would actually fit better under registry/advanced? If the only place it's getting used it for modules, maybe that makes sense? @JessamyT might have thoughts around this

Reply:
I can definitely see it being in registry/advanced. I think that so long as we link to it from all relevant places I don't have strong opinions on where it goes.